src/user-guide/sharing-workflows/sharing-workflows.rst

```diff
@@ -0,0 +1,285 @@
+.. _Authorization:
+
+Authorization
+=============
+
+Cylc supports multi user access. You can grant control of your
+workflows to other users, by adding or removing privileges on a Cylc operation
+basis.This requires site level configuration, with individual users being able
+to set authorization settings, within the bounds set by the site.
+
+Sites can set default access rights for users, for more information, see
+:ref:`site_configuration`.
+
+Note, if this feature is not configured, the default is no multi user access
+and access will be limited to the workflow owner.
+
+If you grant access to another user, this access will apply to all workflows.
+
+Granting Access
+---------------
+
+There are three methods of identifying a user to grant access to:
+
+- ``*`` character, to indicate any authenticated user,
+- using the ``group:`` prefix to indicate a system group. E.g.
+  ``group:groupname``, will assign permissions to all members of the system
+  group named `groupname`. For more information, see :ref:`group_support`.
+- ``username`` to indicate a specific user.
+
+All permissions are additive, if user appears elsewhere in configuration, for
+example as a member of a system group, the permission level is taken as the
+greatest possible.
+
+However, any negations are applied with priority.
+
+Note, default in site config does not contribute to permissions, this is used
+in the case where a user does not appear in the `c.CylcUIServer.user-authorization`.
+
+.. note::
+
+   Spellings of UIServer configurations should be strictly adhered to. Using,
+   for example, `c.CylcUIServer.user-authorisation` rather than
+   `c.CylcUIServer.user-authorization` would result in cylc silently ignoring
+   this configuration.
```

Comment:
I think there is now some form of warning message for invalid configs in Jupyter Server. Will have to try it out to confirm.

Reply:
Ah yes, I remember you mentioning this. I couldn't spot any message but entirely possible I have missed them, unless it is a feature we have to turn on.

Reply:
Here's my test:

`~/.cylc/hub/jupyter_config.py`:
```python
c.CylcUIServer.not_a_trait = False
```

```console
$ cylc gui
[W 2021-10-05 13:10:22.435 CylcUIServer] Config option `not_a_trait` not recognized by `CylcUIServer`.
...
```

Reply:
Ah I have gone through the hub and couldn't spot a warning :) I shall change the docs.

Reply:
Also, I wonder if there is a way of returning a warning if using `cylc hub`?